src/constants/pools.ts

```diff
@@ -388,7 +388,6 @@ const POOLS_POLYGON: Pools = {
     'Gyro2',
     'Gyro3',
     'GyroE',
-    'FX',
```

Comment:
somehow FX pool type was not removed from excluded list, but worked anyway, probably api doesn't handle this

Reply:
in current pool request to api, we don't send excluded pools list, https://github.com/balancer-labs/frontend-v2/blob/33ee35e5fc7ea412e060fa81ec0c218426bff6e1/src/services/balancer/api/entities/pool/index.ts#L37

so it seems we need to change the whole request flow

I remember you wanted to refactor/combine usePoolsQuery and usePoolQuery, I think its better to do in the scope of this refactoring as it will fix this issue